recipes/mosdef-dihedral-fit/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "mosdef-dihedral-fit" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/GOMC-WSU/MoSDeF-dihedral-fit/archive/{{ version }}.tar.gz
+  sha256: 321272e79b1f86d44eef29b10f296edfd60de9faa9e0d4b4551ec4f898e2f69a
+
+build:
+  skip: true  # [win or py<3.10 or py>3.11]
```

Comment:
Why are you skipping Windows? Note it is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
Thanks for letting us know! We were skipping windows because our windows build is failing (and exactly because missing one of our dependency). I will reinstate the noarch build shortly.

Reply:
done!